### PR TITLE
address rename treeview update regression

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Fix the updating of MVS and USS tree when a file or folder is renamed

## Release Notes

Milestone: 2.14.0

Changelog: - Fixed issue seen after renaming an MVS or USS file or folder. [#2656](https://github.com/zowe/vscode-extension-for-zowe/issues/2656)

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [x] There is coverage for the code that I have added
- [x] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Review by adam-wolfe — DISMISSED** [review]
LGTM, thanks @JillieBeanSim

**Review by adam-wolfe — APPROVED** [review]
Current implementation solves the issue. However, it may be desirable to have the tree sort itself after renaming a file/ds.

File: packages/zowe-explorer/CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed failure to refresh token value after user logs in to authentication. [#2638](https://github.com/zowe/vscode-extension-for-zowe/issues/2638)
 - Fixed order of spool files reverses when the Job is expanded and collapsed. [#2644](https://github.com/zowe/vscode-extension-for-zowe/pull/2644)
 - Fixed local filtering of jobs to work with SMFID (exec-member field). [#2651](https://github.com/zowe/vscode-extension-for-zowe/pull/2651)
+- Fixed issue seen after renaming an MVS or USS file or folder. [#2656](https://github.com/zowe/vscode-extension-for-zowe/issues/2656)
```
Comment (t1m0thyj) [inline packages/zowe-explorer/CHANGELOG.md:21]: Should we be more specific here? Something like "Fixed tree item labels failing to update after renaming..."

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @JillieBeanSim!

File: packages/zowe-explorer/CHANGELOG.md (hunk 1 above)
Comment (JillieBeanSim) [reply packages/zowe-explorer/CHANGELOG.md:21]: Hey Timothy, thanks added your suggestion in Prepare 2.14.0 Release [PR](https://github.com/zowe/vscode-extension-for-zowe/pull/2661/files)

**Pull request merged**